comps/comps-foreman-el8.xml

```diff
@@ -238,6 +238,7 @@
       <packagereq type="default">rubygem-hirb</packagereq>
       <packagereq type="default">rubygem-hirb-unicode-steakknife</packagereq>
       <packagereq type="default">rubygem-hoe</packagereq>
+      <packagereq type="default">tfm-rubygem-hocon</packagereq>
```

Comment:
EL8 doesn't use tfm prefixes.

Reply:
Jeez how could have I missed this one? Fixed.